test-suite/squidconf/bug4832.conf

```diff
@@ -1,2 +1,9 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
```

Comment:
Since this PR is fixing broken test-sutie/squidconf/*.conf copyright blobs, please fix the blob in test-suite/squidconf/mgr_passwd.conf (which has an inconsistent comment style).

Reply:
done